src/components/PunsForm/PunsForm.jsx

```diff
@@ -0,0 +1,70 @@
+import "./PunsForm.css";
+
+export const PunsForm = ({
+  date,
+  handleDate,
+  dev,
+  handleDev,
+  context,
+  handleContext,
+  pun,
+  handlePun,
+  handleSubmit,
+}) => {
+  return (
+    <div className="form__wrapper">
+      <form className="form" onSubmit={handleSubmit}>
+        <h2 className="title">Add New Pun</h2>
```

Comment:
why not `form__title` in the className? just `title` seems generic. In your header ur title is `header_title`